Safety/boardfiles/Src/main.cpp

```diff
@@ -169,9 +171,9 @@ int main(void)
 
   // start the IC in interrupt mode
 
-
+  attitudeManager *attMng;
   PPMChannel ppm;
-  ppm.setNumChannels(8);
+  ppm.setNumChannels(MAX_PPM_CHANNELS);
```

Comment:
Is this ppm object still needed? Looks like it was moved to attitudeStateClasses.cpp